library/common/config_template.cc

```diff
@@ -51,8 +51,45 @@ R"(
       endpoints: *base_endpoints
     tls_context: *base_tls_context
     type: LOGICAL_DNS
+  - name: stats
+    connect_timeout: {{ connect_timeout_seconds }}s
+    dns_refresh_rate: {{ dns_refresh_rate_seconds }}s
+    dns_lookup_family: V4_ONLY
+    http2_protocol_options: {}
+    lb_policy: ROUND_ROBIN
+    load_assignment:
+      cluster_name: base_wwan
+      endpoints:
+        - lb_endpoints:
+            - endpoint:
+                address:
+                  socket_address: {address: {{ stats_domain }}, port_value: 443}
+    tls_context: *base_tls_context
+    type: LOGICAL_DNS
 stats_flush_interval: {{ stats_flush_interval_seconds }}s
+stats_sinks:
+  - name: envoy.metrics_service
+    config:
+      grpc_service:
+        envoy_grpc:
+          cluster_name: stats
+stats_config:
+  stats_matcher:
+    inclusion_list:
```

Comment:
initial testing metrics. I wanted to include a counter, a gauge, and a histogram. We can discuss what we want to include, and also if we want to make this an  injectable list.